server/utils/userUtils.ts

```diff
@@ -34,6 +35,13 @@ export const sections = {
     shortTitle: 'Management information reports',
     href: reportsPaths.report.new.pattern,
   },
+  prisonApplications: {
+    id: 'prison-dashboard',
```

Comment:
FYI The ID needs updating to 'prison-applications', I'll wait until it's reviewed to change.